docs/AWS_Integration_Guide/Configuring_AWS_integration.md

```diff
@@ -102,6 +83,8 @@ To add the resources needed for the integration to your AWS services:
     ```
     python3 -m pip install -r lwm2mOperation/requirements.txt --target lwm2mOperation/
     python3 -m pip install -r certificates/requirements.txt --target certificates/
+    python3 -m pip install -r createThingType/requirements.txt --target createThingType/
```

Comment:
I see the requirements were added but I don't see the two new lambdas, do I understand correctly that it should be included in (some) revision?

Reply:
look inside your heart. maybe the true lambdas were the friends you made along the way?